### PR TITLE
Updated the README file to include _itol_metadata.py changes

### PR DESCRIPTION
**Pull request by stephramos17**
Updated the README documentation to include instructions on how to use the _itol_metadata.py file to generate barcharts in the tree with metadata group/condition information.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -190,11 +190,29 @@ Users can choose one of the following data columns (`--p-column`) for pruning: '
...
+When `--color-file-path` and `--label-file-path` is not specified by the user, this command generates two files: 'itol_colors.txt' and 'itol_labels.txt' in place. One can upload the tree generated above in [iTOL](https://itol.embl.de/), and drag & drop these two files to 1) color tree clades based on the specified Classyfire level ('subclass' here) 2) label tree tips by the Classyfire category they belong to. For easier interpretativity, setting the `--ms2-label` as False labels all the tree tips based on the CSI:FingerID prediction, and not the MS/MS library matches. This enables the users to visualize the chemical diversity in their samples and better understand the underlying chemistry.
```
Comment (anupriyatripathi) [inline README.md]: The most updated version does not generate these files by default unless the user enters a path. This avoids accumulations of junk files if not needed. Could you please update this as well?

File: README.md [hunk 2]
```diff
@@ -190,11 +190,29 @@ Users can choose one of the following data columns (`--p-column`) for pruning: '
...
+python _itol_metadata.py \
+  --classified-feature-data classified-merged-feature-data.qza \
+  --feature-data-column subclass \
+  --ms2-label False \
+  --color-file-path path-to-clade-colors-file.txt \
+  --label-file-path path-to-tip-label-files.txt \
+  --feature-table feature-table-hashed.qza \
+  --sample-metadata metadata.tsv \
+  --sample-metadata-column groups \
+  --barchart-file-path path-to-barchart-subclass-file
```
Comment (anupriyatripathi) [inline README.md]: Could we just show the command where user could just generate a file for bar charts without giving the color and label path? You can also specify this is the text that one can just generate a subset of these metadata files based on whatever is needed.

Comment (stephramos17) [reply README.md]: ok cool! I addressed all the comments and made the changes, do I have to submit a new push request with the changes? Or is it already cataloged here?

File: README.md [hunk 3]
```diff
@@ -140,7 +140,7 @@ Additionally, Qemistree also supports the inclusion of structural annotations ma
 qiime qemistree make-hierarchy \
   --i-csi-results fingerprints.qza \
   --i-feature-tables feature-table.qza \
-  --i-ms2-matches /path-to-MS2-spectral-matches.qza/ \
+  --i-ms2-matches path-to-MS2-spectral-matches.qza/ \
```
Comment (anupriyatripathi) [inline README.md]: I think the previous version is fine for this. It's just convention to write /path-to-blah/ inside forward slashes.

File: README.md [hunk 4]
```diff
@@ -150,7 +150,10 @@ qiime qemistree make-hierarchy \
 1. The input to `--i-ms2-matches` can be obtained using [Feature-based molecular networking or FBMN](https://gnps.ucsd.edu/ProteoSAFe/index.jsp?params=%7B%22workflow%22:%22FEATURE-BASED-MOLECULAR-NETWORKING%22,%22library_on_server%22:%22d.speclibs;%22%7D) workflow supported in the web-based mass-spectrometry data analysis platform, [GNPS](https://gnps.ucsd.edu/). To use MS2 matches in Qemistree, please download the results of FBMN workflow and import the tsv file in the folder `DB_result` as a QIIME2 artifact of type `FeatureData[Molecules]` as follows:
 
 ```bash
-qiime tools import --input-path path-to-MS2-spectral-matches.tsv --output-path path-to-MS2-spectral-matches.qza --type FeatureData[Molecules]
+qiime tools import \
+  --input-path path-to-MS2-spectral-matches.tsv \
+  --output-path path-to-MS2-spectral-matches.qza \
+  --type FeatureData[Molecules]
```
Comment (anupriyatripathi) [inline README.md]: Thanks 👍

File: README.md [hunk 5]
```diff
@@ -190,11 +193,26 @@ Users can choose one of the following data columns (`--p-column`) for pruning: '
...
+python _itol_metadata.py \
+  --classified-feature-data classified-merged-feature-data.qza \
+  --feature-data-column subclass \
+  --feature-table feature-table-hashed.qza \
+  --sample-metadata metadata.tsv \
+  --sample-metadata-column groups \
+  --barchart-file-path path-to-barchart-subclass-file
+```
+
+The `path-to-barchart-subclass-file.qza` file generated can also be dragged & dropped into [iTOL](https://itol.embl.de/).
```
Comment (anupriyatripathi) [inline README.md]: The following should do it. See comment above for convention about representing paths.

/path-to-barchart.qza/

Comment (stephramos17) [reply README.md]: Sounds good! I change all the code chunks to have the right path convention

**Pull request merged**